ci/tasks/node-build.yml

```diff
@@ -14,7 +14,7 @@ outputs:
   - name: build-output
 
 caches:
-  - path: src/node_modules
+  - path: ../../../root/.npm
```

Comment:
Just wondering if this is susceptible to changing under our feet since it's outside the build environment (from what I understand). Maybe something to think about for the future.